index.html

```diff
@@ -128,14 +135,12 @@ <h3 class="works-card-title">Multi-Post Stories Gain+Glory</h3>
         <!-- About  section -->
         <section id="about-section">
             <div class="section-container">
-                <div class="about-title">
+                <div class="section-title">
                     <h2>About Me</h2>
-                    <p class="supporting-text">I can help you build a product,<br>
-                        feature or website Look through <br>
-                        some of my work and experience! If<br>
-                        you like what you see and have a <br>
-                        project you need coded,<br>
-                        don’t hesitate to contact me.
+                    <p class="supporting-text">
+                        I can help you build a product, feature or website. Look through some of my work and <br>
+                        experience! If you like what you see and have <br>a project you need coded, <br>don't hesitate
+                        to contact me.
```

Comment:
- [ ] You have done a great job here. :+1: :+1:, But you still need to make some improvements before moving to the next milestone. :smile: 
- [ ] As part of the requirement for the desktop version is no make your design look like the Figma design as much as possible, I think your tex placement doesn't match the one in the Figma design. I'll suggest you look into this and align your text properly to the center. :+1: :+1: 
- - Your design:
![Screenshot from 2023-05-11 23-16-02](https://github.com/RileyManda/portfolio-website/assets/107632780/1ef9a208-c5d8-4551-b32e-656377a38381)

- - Figma design:
![Screenshot from 2023-05-11 23-16-22](https://github.com/RileyManda/portfolio-website/assets/107632780/6c0223aa-1c82-4961-8347-a46757843864)